### DISCUSSION TITLE
Does setting "MARQO_MAX_NUMBER_OF_REPLICAS=x", creates 'x' replicas of each index?

### DISCUSSION THREAD

**Discussion by renuka010** [marqo-ai/marqo#876] [Q&A] [answered]
In Documentation it is mentioned MARQO_MAX_NUMBER_OF_REPLICAS=2 create maximum of 2 replicas. Does this mean it creates 2 copies of the data I am indexing always?

**Comment by papa99do** [comment, marked as answer]
Hello @renuka010 , thanks for bringing this up. I checked the code base and can confirm that this env var is removed in Marqo 2.x. This env var is used to control the maximum number of replicas in open_search in Marqo 1.x, and becomes invalid when we move on to Vespa in Marqo 2.x. We will update the document to remove this. 

On the other hand, if you would like to achieve HA in your setup, you'll need to have an external Vespa cluster deployed and configure the replica/redundancies properly. Please see [this doc](https://docs.vespa.ai/en/elasticity.html) for details. If you are using k8s, you can also follow this repo: https://github.com/marqo-ai/marqo-on-kubernetes

Please let me know if this answers you questions. Cheers.

**Reply by renuka010** [reply]
Yes this is helpful @papa99do
